front-end-node/profiler/SaveOverrides.js

```diff
@@ -0,0 +1,38 @@
+/*jshint browser:true, nonew:false*/
+/*global WebInspector:true*/
+
+WebInspector.saveAs = function(blob,filename){
+  var url = URL.createObjectURL(blob);
+  var link = document.createElementNS("http://www.w3.org/1999/xhtml", "a");
+  link.download = filename;
+  link.href = url;
+  var event = document.createEvent("MouseEvents");
+  event.initMouseEvent(
+    "click", true, false, window, 0, 0, 0, 0, 0
+    , false, false, false, false, 0, null
+  );
+  link.dispatchEvent(event);
```

Comment:
I saw it in FileSaver but it's deprecated according to MDN https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/initMouseEvent

I'll try `link.click()` or switch to proper event dispatching for Chromium project...